### PR TITLE
Add support for other trivial `Put*` Bucket attributes

### PR DESCRIPTION
**Pull request by RedbackThomson**
Adds support for the following Bucket attributes:
- AccelerateConfiguration
- CORS
- Encryption
- OwnershipControls
- RequestPayment
- Tagging
- Website

Updates for the following attributes are not supported (see https://github.com/aws-controllers-k8s/community/issues/757):
- CORS
- Encryption
- OwnershipControls
- Tagging

### CODE REVIEW

File: apis/v1alpha1/ack-generate-metadata.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-08-02T22:37:09Z"
+  build_date: "2021-08-03T01:38:05Z"
   build_hash: 8b9ae5dfab674f9d759bf6ccb1aef88bc91f2e4f
   go_version: go1.15.6 linux/amd64
   version: v0.8.0
-api_directory_checksum: f3302d83fa5db4156550a01a1d76ba8e3509386b
+api_directory_checksum: a204a5150e6c86b28b6da5d5ffbe934b0aa11018
 api_version: v1alpha1
 aws_sdk_go_version: v1.37.4
```
Comment (vijtrip2) [inline apis/v1alpha1/ack-generate-metadata.yaml:8]: should this be 1.37.10?

File: apis/v1alpha1/ack-generate-metadata.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-08-02T22:37:09Z"
+  build_date: "2021-08-03T01:38:05Z"
   build_hash: 8b9ae5dfab674f9d759bf6ccb1aef88bc91f2e4f
   go_version: go1.15.6 linux/amd64
   version: v0.8.0
```
Comment (vijtrip2) [inline apis/v1alpha1/ack-generate-metadata.yaml:5]: Should this be updated to v0.9.0?

**Review by vijtrip2 — COMMENTED** [review]
No need to update e2e tests ??

File: apis/v1alpha1/ack-generate-metadata.yaml (hunk 2 above)
Comment (RedbackThomson) [reply apis/v1alpha1/ack-generate-metadata.yaml:5]: I haven't yet updated to `0.9.0` - not for this PR at least.

File: apis/v1alpha1/ack-generate-metadata.yaml (hunk 1 above)
Comment (RedbackThomson) [reply apis/v1alpha1/ack-generate-metadata.yaml:8]: I've pinned S3 to `1.37.4` for now - https://github.com/aws-controllers-k8s/s3-controller/blob/main/go.mod#L7

Although I wouldn't expect there to be any changes in the API between this and `1.37.10`, so probably safe to update.

File: apis/v1alpha1/generator.yaml [hunk 3]
```diff
@@ -35,14 +35,10 @@ resources:
...
       #     path: InventoryConfiguration # Double check about ID
-      # # Lifecycle:
-      # #   from:
-      # #     operation: PutBucketLifecycle
-      # #     path: LifecycleConfiguration
       # LifecycleConfiguration:
       #   from:
       #     operation: PutBucketLifecycleConfiguration
-      #     path: LifecycleConfiguration
+      #     path: LifecycleConfiguration # Double check weird types
```
Comment (jaypipes) [inline apis/v1alpha1/generator.yaml:41]: :)

File: generator.yaml [hunk 4]
```diff
@@ -11,22 +11,22 @@ operations:
 resources:
   Bucket:
     fields:
-      # AccelerateConfiguration:
-      #   from:
-      #     operation: PutBucketAccelerateConfiguration
-      #     path: AccelerateConfiguration
+      AccelerateConfiguration:
```
Comment (jaypipes) [inline generator.yaml:14]: Would be cool if we named this just Accelerate to aling with the way that CORS, Logging, Encryption and Website fields work...

File: go.local.mod [hunk 5]
```diff
@@ -5,8 +5,8 @@ go 1.14
 replace github.com/aws-controllers-k8s/runtime => ../runtime
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.7.1
-	github.com/aws/aws-sdk-go v1.37.4
+	github.com/aws-controllers-k8s/runtime v0.8.0
```
Comment (jaypipes) [inline go.local.mod:8]: go.local.mod changes to runtime are ignored (because of the `replace` line above...)

**Review by jaypipes — APPROVED** [review]
Looks good to me. I presume tests are coming in a followup PR, yes?

**Comment by RedbackThomson** [comment]
> Looks good to me. I presume tests are coming in a followup PR, yes?

Sure

**Pull request merged**